lms/envs/common.py

```diff
@@ -45,7 +45,7 @@
 
 ################################### FEATURES ###################################
 # The display name of the platform to be used in templates/emails/etc.
-PLATFORM_NAME = "Your Platform Name Here"
+PLATFORM_NAME = _('edX')
```

Comment:
Every ocurrence of `edX` is a big thing, but settings.PLATFORM_NAME is used across the platform a lot. I would not like that this reverts back to 'edX' by default